packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab/components/SocialConnectorEditBox/AddSocialConnectorButton.module.scss

```diff
@@ -0,0 +1,37 @@
+@use '@/scss/underscore' as _;
+
+.dropdown {
+  min-width: 208px;
+}
+
+.plusIcon {
+  color: var(--color-text-secondary);
+}
+
+.title {
+  display: flex;
+  align-items: center;
+
+  .logo {
+    margin-right: _.unit(3);
+    width: 20px;
+    height: 20px;
+
+    img {
+      width: 20px;
+      height: 20px;
+    }
+  }
+
+  .name {
+    font: var(--font-body-2);
```

Comment:
we don't have `--font-body-2` now, please check the new font tokens. Try `--font-body-mediem` maybe?

Reply:
Cool, updated.